File: internal/http/file/const.go
package file

const filesFolder = "stock"
const maxFilesAmount = 10
const fileFieldName = "file"

var maxFileSizes = map[string]int64{
	"images": 2 * 1024 * 1024,  // 2 MB
	"audios": 5 * 1024 * 1024,  // 5 MB
	"videos": 25 * 1024 * 1024, // 25 MB
	"files":  10 * 1024 * 1024, // 10 MB
}

var knownContentTypes = map[string]string{
	"image/gif":     "images",
	"image/jpeg":    "images",
	"image/png":     "images",
	"image/svg+xml": "images",
	"image/webp":    "images",

	"video/mp4":       "videos",
	"video/webm":      "videos",
	"video/ogg":       "videos",
	"video/quicktime": "videos",
	"video/x-flv":     "videos",

	"audio/mpeg": "audios",
	"audio/ogg":  "audios",
	"audio/wav":  "audios",
	"audio/aac":  "audios",
}

File: web/src/modules/chat/const.ts
import { AttachmentType } from "./types";

export const BOT_COMMAND_REGEX = /\/[a-z0-9_\-]+(?= |$)/gm;
export const NOT_HTML_NEWLINE_REXEG = /^(?![^<]*<\/[^>]*>)\n/gm;

export const KNOWN_MIME_TYPES: { [key: string]: AttachmentType } = {
    "image/gif": "image",
    "image/jpeg": "image",
    "image/png": "image",
    "image/svg+xml": "image",
    "image/webp": "image",

    "video/mp4": "video",
    "video/webm": "video",
    "video/ogg": "video",
    "video/quicktime": "video",
    "video/x-flv": "video",

    "audio/mpeg": "audio",
    "audio/ogg": "audio",
    "audio/wav": "audio",
    "audio/aac": "audio",
};
